BUILDING.md

```diff
@@ -0,0 +1,68 @@
+## Build the development environment
+
+The first prerequisite of properly building SwarmKit targets is to have a Go
+development environment setup. Please follow [How to Write Go Code](https://golang.org/doc/code.html)
+for proper setup. If done correctly, you should have a GOROOT and GOPATH set in the
+environment.
+
+If a Go development environment is setup, one can use `go get` to install the
```

Comment:
After you set up the Go development environment, use `go get` to install `swarmkit`: